metadata-models/docs/entities/assertion.md

```diff
@@ -0,0 +1,37 @@
+# Assertion
+
+Assertion entity represents a data quality rule applied on dataset.  
+In future, it can evolve to span across Datasets, Flows (Pipelines), Models, Features etc.
+
+## Identity
+
+Assertion is identified by globally unique identifier, which is function of details that uniquely identify the assertion in an assertion platform.
+
+## Important Capabilities
```

Comment:
Can we name this section as "Aspects"?

Reply:
Let's be consistent across the board for these sections...  Datasets has `Important Capabilities` ... maybe we should call them `Features` or something else.. the goal was to separate this from the lower-level Aspects section which is auto-generated.